apis/r/R/RcppExports.R

```diff
@@ -92,9 +92,9 @@ check_arrow_array_tag <- function(xp) {
 #' uri <- "test/soco/pbmc3k_processed/obs"
 #' sr <- sr_setup(uri, config=as.character(config(ctx)), loglevel="warn")
 #' rl <- data.frame()
-#' while (nrow(rl) == 0 || !tiledbsoma:::sr_complete(sr)) {
-#'     dat <- tiledbsoma:::sr_next(sr)
-#'     dat |>
+#' while (nrow(rl) == 0 || !sr_complete(sr)) {
```

Comment:
The `nrow(rl) == 0` is no longer needed.  Also, when changing the roxygen stuff please edit only in the _C++ source_ file, use `compileAttributes()` to get it here and `roxygenize()` to create the .Rd file.

Reply:
done